### PR TITLE
Tooling v6

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: doc/tuto/start.md [hunk 1]
```diff
@@ -21,15 +21,53 @@ see something like this:
...
 All the scenes listed on the left tree view can be found in the
 [pynodegl_utils.examples][demo-tree] Python module. If you are curious on how
 each demo scene operates, look into this place.
 
+Also note that `ngl-control` supports live code editing by monitoring the files
+associated with the available scenes. The current scene will be reconstructed
+after every change in the sources.
+
+On the top-left part of the controller, you can control the generic parameters
+sent to each scene when they are constructed. Try to play with the them and see
```
Comment (mbouron-gpsw) [inline doc/tuto/start.md:54]: Typo: with the them

Comment (cboesch-gpsw) [reply doc/tuto/start.md]: Fixed

File: doc/tuto/start.md [hunk 2]
```diff
@@ -21,15 +21,53 @@ see something like this:
 
 ![ngl-control](img/ngl-control.png)
 
+`ngl-control` doesn't render scenes directly, it's a controller which builds
+scenes from Python sources, then communicates the result to `ngl-desktop`, the
```
Comment (mbouron-gpsw) [inline doc/tuto/start.md:25]: maybe: then communicates the result to `ngl-desktop` for rendering. `ngl-desktop` is our native rendering player on desktop.

**Review by mbouron-gpsw — APPROVED** [review]
Other than my remarks, LGTM.

File: doc/tuto/start.md (hunk 2 above)
Comment (cboesch-gpsw) [reply doc/tuto/start.md]: Took your suggestion, just replaced "our" with "the".

**Pull request merged**